election-ui-javascript/src/components/Scoreboard.test.js

```diff
@@ -1,32 +1,28 @@
-import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import { fireEvent, render, screen, waitFor, within } from '@testing-library/react';
 import Scoreboard from './Scoreboard';
 import dataFetcher from '../dataFetcher';
 
 jest.mock('../dataFetcher');
 
-const mockDataFetcher = () => {
+test('renders Results', async () => {
   dataFetcher.mockImplementationOnce(() => {
     return Promise.resolve({
       isComplete: false,
       results: [
         {
-          'party': 'Independent',
+          'party': 'Giraffe Party',
           'candidateId': 2,
           'votes': '9900'
         }
       ]
     })
   });
-}
-
-test('renders Results', async () => {
-  mockDataFetcher();
 
   render(<Scoreboard />);
 
   await waitFor(() => {
-    const results = screen.getByText(/Independent/i);
-    expect(results).toBeInTheDocument();
+    const resultParty = within(screen.getByRole('table')).getByText(/Giraffe Party/i);
```

Comment:
Now that we have the `<PartyLinks />`, the party name actually appears twice in the document. So we can't just narrow this down with a plain `.getByText()` anymore 😢